# HG changeset patch
# User Rik <rik@octave.org>
# Date 1541444830 28800
#      Mon Nov 05 11:07:10 2018 -0800
# Node ID 0a39db17170c35a136a1609c9c1a4b55ec608908
# Parent  093ee7420280dd53d67fb7461c213a89541fa562
doc: Clean up docstrings for XXXdlg.m functions

* errordlg.m, helpdlg.m, inputdlg.m, listdlg.m, msgbox.m, uicontrol.m,
warndlg.m: Use TexInfo macro @: to stop interpretation of '.' in message
as an end-of-sentence period.  Correct typo 'optionnal' -> 'optional'.
Wrap long lines.  Correct typo 'elment' -> 'element'.  Wrap "keypressfcn"
in @qcode macro.

diff --git a/scripts/gui/errordlg.m b/scripts/gui/errordlg.m
--- a/scripts/gui/errordlg.m
+++ b/scripts/gui/errordlg.m
@@ -20,23 +20,23 @@
 ## @deftypefn  {} {} errordlg ()
 ## @deftypefnx {} {} errordlg (@var{msg})
 ## @deftypefnx {} {} errordlg (@var{msg}, @var{title})
 ## @deftypefnx {} {} errordlg (@var{msg}, @var{title}, @var{opt})
 ## @deftypefnx {} {@var{h} =} errordlg (@dots{})
 ## Display an error dialog box with error message @var{msg} and caption
 ## @var{title}.
 ##
-## The default error message is @qcode{"This is the default error string."} and
-## the default caption is @qcode{"Error Dialog"}.
+## The default error message is @qcode{"This is the default error string.@:"}
+## and the default caption is @qcode{"Error Dialog"}.
 ##
 ## The error message may have multiple lines separated by newline characters
 ## ("\n"), or it may be a cellstr array with one element for each line.
 ##
-## The third optionnal argument @var{opt} controls the behavior of the dialog.
+## The third optional argument @var{opt} controls the behavior of the dialog.
 ## See @code{msgbox} for details.
 ##
 ## The return value @var{h} is a handle to the figure object used for
 ## building the dialog.
 ##
 ## Examples:
 ##
 ## @example
diff --git a/scripts/gui/helpdlg.m b/scripts/gui/helpdlg.m
--- a/scripts/gui/helpdlg.m
+++ b/scripts/gui/helpdlg.m
@@ -19,18 +19,18 @@
 ## -*- texinfo -*-
 ## @deftypefn  {} {} helpdlg ()
 ## @deftypefnx {} {} helpdlg (@var{msg})
 ## @deftypefnx {} {} helpdlg (@var{msg}, @var{title})
 ## @deftypefnx {} {@var{h} =} helpdlg (@dots{})
 ## Display a help dialog box with help message @var{msg} and caption
 ## @var{title}.
 ##
-## The default help message is @qcode{"This is the default help string."} and
-## the default caption is @qcode{"Help Dialog"}.
+## The default help message is @qcode{"This is the default help string.@:"}
+## and the default caption is @qcode{"Help Dialog"}.
 ##
 ## The help message may have multiple lines separated by newline characters
 ## ("\n"), or it may be a cellstr array with one element for each line.
 ##
 ## The return value @var{h} is a handle to the figure object used for
 ## building the dialog.
 ##
 ## Examples:
diff --git a/scripts/gui/inputdlg.m b/scripts/gui/inputdlg.m
--- a/scripts/gui/inputdlg.m
+++ b/scripts/gui/inputdlg.m
@@ -35,25 +35,30 @@
 ## @item title
 ## String to use for the caption of the dialog.  The default is
 ## @qcode{"Input Dialog"}.
 ##
 ## @item rowscols
 ## Specifies the size of the text fields and can take three forms:
 ##
 ## @enumerate
-## @item a scalar value which defines the number of rows used for each text field.
+## @item a scalar value which defines the number of rows used for each text
+## field.
 ##
-## @item a vector which defines the individual number of rows used for each text field.
+## @item a vector which defines the individual number of rows used for each
+## text field.
 ##
-## @item a matrix which defines the individual number of rows and columns used for each text field.  In the matrix each row describes a single text field.  The first column specifies the number of input rows to use and the second column specifies the text field width.
+## @item a matrix which defines the individual number of rows and columns used
+## for each text field.  In the matrix each row describes a single text field.
+## The first column specifies the number of input rows to use and the second
+## column specifies the text field width.
 ## @end enumerate
 ##
 ## @item defaults
-## A list of default values to place in each text fields.  It must be a cell
+## A list of default values to place in each text field.  It must be a cell
 ## array of strings with the same size as @var{prompt}.
 ##
 ## @item options
 ## Not supported, only for @sc{matlab} compatibility.
 ## @end table
 ##
 ## Example:
 ##
@@ -224,21 +229,21 @@ endfunction
 %!   helpdlg (sprintf ('Results:\nVolume = %.3f\nSurface = %.3f', ...
 %!                     volume, surface), 'Box Dimensions');
 %! endif
 
 %!demo
 %! disp ('- test inputdlg with vector for a single item.');
 %! prompt = {'enter x value'};
 %! default = {1};
-%! answer = inputdlg (prompt,'Enter value', [1 10], default);
+%! answer = inputdlg (prompt, 'Enter value', [1 10], default);
 %! if (isempty (answer))
 %!   helpdlg ('Canceled by user', 'Information');
 %! else
-%!   helpdlg (sprintf ('answer = %d', str2num(answer{1})), 'answer');
+%!   helpdlg (sprintf ('answer = %d', str2num (answer{1})), 'answer');
 %! endif
 
 %!error inputdlg (1, 2, 3, 4, 5, 6)
 %!error <PROMPT must be a character string> inputdlg (1)
 %!error <TITLE must be a character string> inputdlg ("msg", 1)
 %!error <ROWSCOLS must be numeric> inputdlg ("msg", "title", "1")
 %!error <ROWSCOLS vector does not match size>
 %! inputdlg ({"a1", "a2"}, "title", [1, 2, 3]);
diff --git a/scripts/gui/listdlg.m b/scripts/gui/listdlg.m
--- a/scripts/gui/listdlg.m
+++ b/scripts/gui/listdlg.m
@@ -36,17 +36,17 @@
 ## @item @qcode{"ListString"}
 ## a cell array of strings specifying the items to list in the dialog.
 ##
 ## @item @qcode{"SelectionMode"}
 ## can be either @qcode{"Single"} (only one item may be selected at a time) or
 ## @qcode{"Multiple"} (default).
 ##
 ## @item @qcode{"ListSize"}
-## a two-elment vector @code{[@var{width}, @var{height}]} specifying the size
+## a two-element vector @code{[@var{width}, @var{height}]} specifying the size
 ## of the list field in pixels.  The default is [160, 300].
 ##
 ## @item @qcode{"InitialValue"}
 ## a vector containing 1-based indices of elements which will be pre-selected
 ## when the list dialog is first displayed.
 ## The default is 1 (first item).
 ##
 ## @item @qcode{"Name"}
diff --git a/scripts/gui/msgbox.m b/scripts/gui/msgbox.m
--- a/scripts/gui/msgbox.m
+++ b/scripts/gui/msgbox.m
@@ -35,40 +35,40 @@
 ## It can be one of @qcode{"none"} (default), @qcode{"error"}, @qcode{"help"},
 ## @qcode{"warn"}, or @qcode{"custom"}.  The latter must be followed by an
 ## image array @var{cdata}, and for indexed images the associated colormap.
 ##
 ## The final optional argument @var{opt} controls the behavior of the dialog.
 ## If @var{opt} is a string, it may be one of
 ##
 ## @table @asis
+## @item @qcode{"non-modal"} (default)
+## The dialog is normal.
+##
 ## @item @qcode{"modal"}
 ## The dialog is displayed @qcode{"modal"} which means it prevents users from
-## interacting with any other GUI element.
-##
-## @item @qcode{"non-modal"} (default)
-## The dialog is normal.
+## interacting with any other GUI element until the dialog has been closed.
 ##
 ## @item @qcode{"replace"}
 ## If any dialogs already exist with the same title, the most recent is reused
 ## and all others are closed.  The resulting dialog is set @qcode{"non-modal"}.
 ## @end table
 ##
 ## If @var{opt} is a structure, it must contain fields @qcode{"WindowStyle"}
 ## and @qcode{"Interpreter"}:
 ##
 ## @table @asis
 ## @item @qcode{"WindowStyle"}
-## The value must be @qcode{"modal"}, @qcode{"non-modal"}, or
+## The value must be @qcode{"non-modal"}, @qcode{"modal"}, or
 ## @qcode{"replace"}.  See above.
 ##
 ## @item @qcode{"Interpreter"}
 ## Controls the @qcode{"interpreter"} property of the text object used for
-## displaying the message.  The value must be @qcode{"none"}, @qcode{"tex"}
-## (default), or @qcode{"latex"}.
+## displaying the message.  The value must be @qcode{"tex"} (default),
+## @qcode{"none"}, or @qcode{"latex"}.
 ## @end table
 ##
 ## The return value @var{h} is a handle to the figure object used for building
 ## the dialog.
 ##
 ## Examples:
 ##
 ## @example
diff --git a/scripts/gui/uicontrol.m b/scripts/gui/uicontrol.m
--- a/scripts/gui/uicontrol.m
+++ b/scripts/gui/uicontrol.m
@@ -94,17 +94,17 @@
 ## c1 = uicontrol (f, "style", "checkbox", "string", "a checkbox", ...
 ##                    "position", [10 120 150 40]);
 ## @end group
 ## @end example
 ##
 ## When called with a single argument @var{h} which is a handle to an existing
 ## uicontrol object, switch the keyboard focus to the specified
 ## uicontrol.  As a result, the uicontrol object will receive keyboard
-## events that can be processed using the "keypressfcn" callback.
+## events that can be processed using the @qcode{"keypressfcn"} callback.
 ## @seealso{figure, uipanel}
 ## @end deftypefn
 
 ## Author: goffioul
 
 function hui = uicontrol (varargin)
 
   if (nargin == 1 && isgraphics (varargin{1}, "uicontrol"))
diff --git a/scripts/gui/warndlg.m b/scripts/gui/warndlg.m
--- a/scripts/gui/warndlg.m
+++ b/scripts/gui/warndlg.m
@@ -20,23 +20,24 @@
 ## @deftypefn  {} {} warndlg ()
 ## @deftypefnx {} {} warndlg (@var{msg})
 ## @deftypefnx {} {} warndlg (@var{msg}, @var{title})
 ## @deftypefnx {} {} warndlg (@var{msg}, @var{title}, @var{opt})
 ## @deftypefnx {} {@var{h} =} warndlg (@dots{})
 ## Display a warning dialog box with warning message @var{msg} and caption
 ## @var{title}.
 ##
-## The default warning message is @qcode{"This is the default warning string."}
-## and the default caption is @qcode{"Warning Dialog"}.
+## The default warning message is
+## @qcode{"This is the default warning string.@:"} and the default caption is
+## @qcode{"Warning Dialog"}.
 ##
 ## The warning message may have multiple lines separated by newline characters
 ## ("\n"), or it may be a cellstr array with one element for each line.
 ##
-## The third optionnal argument @var{opt} controls the behavior of the dialog.
+## The third optional argument @var{opt} controls the behavior of the dialog.
 ## See @code{msgbox} for details.
 ##
 ## The return value @var{h} is a handle to the figure object used for
 ## building the dialog.
 ##
 ## Examples:
 ##
 ## @example
